src/main/java/seedu/address/logic/commands/listcommands/ListProjectMembersCommand.java

```diff
@@ -34,31 +34,26 @@ public ListProjectMembersCommand(AssignmentProjectPredicate predicate) {
     public CommandResult execute(Model model) {
         requireNonNull(model);
 
-        // Step 1: Filter the assignments based on the predicate
         model.updateFilteredAssignmentList(predicate);
```

Comment:
Be careful that this may cause side-effects of updating the displayed `assignmentList`